### PR TITLE
Introduce edX Certificates (OER-115)

### PR DESCRIPTION
**Pull request by ahmedaljazzar**
### Description

~TASK: [Implement Certificate Authenticity Verification on Edraak (edx-platform)](https://app.asana.com/0/35717934599877/439970783997376/f)~ [DELETED]
TASKS: [Jira tasks](https://edraak.atlassian.net/browse/BUG-65?jql=project%20%3D%20BUG%20AND%20resolution%20%3D%20Unresolved%20AND%20text%20~%20%22certificate%22%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC)

- Introduce Arabic and English Full names.
- Introduce Organizations.
- Removed Edraak link.
- Full names tracking support.
- Added edX Button.
- Some CSS Changes.

Related PRs:
- [Configuration#29](https://github.com/Edraak/configuration/pull/29)
- [Theme#14](https://github.com/Edraak/edraak-2015-theme/pull/14)

### CODE REVIEW

**Comment by ahmedaljazzar** [comment]
@Salomari1987, The pdf version is ready for review.

File: lms/djangoapps/edraak_certificates/edraakcertificate.py [hunk 1]
```diff
@@ -104,68 +62,89 @@ def contains_rtl_text(string):
...
+        self.size = landscape(A4)
+
+        # TODO: Look into this
         ctx = canvas.Canvas(self.temp_file.name)
-        ctx.setPageSize(SIZE)
+        ctx.setPageSize(self.size)
         self.ctx = ctx
 
-    def bidi_x_axis(self, x):
+    def bidi_x_axis(self, x, difference=0):
```
Comment (Salomari1987) [inline lms/djangoapps/edraak_certificates/edraakcertificate.py]: name it offset

File: lms/djangoapps/edraak_certificates/edraakcertificate.py [hunk 2]
```diff
@@ -300,60 +372,141 @@ def save(self):
...
-    def course_org_disclaimer(self):
-        if self.course_org == 'MITX':
-            return self._("A course of study offered by Edraak with cooperation from MITx. "
-                          "The learning experience has been supervised and managed by the course team.")
+    def draw_debugging_grid(self):
+        """
+        This is draws a grid on the certificate if the platform
+        running on a debug mode.
+        """
+        if settings.DEBUG:
```
Comment (Salomari1987) [inline lms/djangoapps/edraak_certificates/edraakcertificate.py]: `if not`

File: lms/templates/certificates/_accomplishment-banner.html [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <%!
```
Comment (Salomari1987) [inline lms/templates/certificates/_accomplishment-banner.html]: don't do it inline, please move it to theme

File: lms/urls.py [hunk 4]
```diff
@@ -969,6 +969,7 @@
         settings.PROFILE_IMAGE_BACKEND['options']['base_url'],
         document_root=settings.PROFILE_IMAGE_BACKEND['options']['location']
     )
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (Salomari1987) [inline lms/urls.py]: please check with ali, we should have access to media

File: lms/djangoapps/certificates/views/webview.py [hunk 5]
```diff
@@ -131,13 +136,10 @@ def _update_certificate_context(context, user_certificate, platform_name):
     # Translators: This text describes the purpose (and therefore, value) of a course certificate
     # 'verifying your identity' refers to the process for establishing the authenticity of the student
     context['certificate_info_description'] = _("{platform_name} acknowledges achievements through certificates, which "
-                                                "are awarded for various activities {platform_name} students complete "
-                                                "under the <a href='{tos_url}'>{platform_name} Honor Code</a>.  Some "
-                                                "certificates require completing additional steps, such as "
-                                                "<a href='{verified_cert_url}'> verifying your identity</a>.").format(
+                                                "are awarded for various activities. {platform_name} students complete "
```
[the remark this answers is not in the document]
Comment (ahmedaljazzar) [reply lms/djangoapps/certificates/views/webview.py]: Will fix it once I receive the official wording from @Edraak/quality-assurance

**Pull request closed without merging**